opencti-platform/opencti-front/src/private/components/analysis/malware_analyses/MalwareAnalysis.tsx

```diff
@@ -0,0 +1,155 @@
+                node {
+                    id
+                    name
+                    entity_type
+                }
+            }
+        }
+        status {
+            id
+            order
+            template {
+                name
+                color
+            }
+        }
+        workflowEnabled
+        ...MalwareAnalysisDetails_malwareAnalysis
+    }
+`;
+
+const MalwareAnalysis: FunctionComponent<{ data: MalwareAnalysis_malwareAnalysis$key }> = ({ data }) => {
+  const classes = useStyles();
+  const malwareAnalysisFragment = useFragment(MalwareAnalysisFragment, data);
+
+  return (
+      <div>
+          <StixDomainObjectHeader
+              stixDomainObject={malwareAnalysisFragment}
+              PopoverComponent={<MalwareAnalysisPopover id={malwareAnalysisFragment.id}/>}
+          />
+          <Grid
+              container={true}
+              spacing={3}
+              classes={{ container: classes.gridContainer }}
+          >
+              <Grid item={true} xs={6} style={{ paddingTop: 10 }}>
+                  <MalwareAnalysisDetails malwareAnalysisFragment={malwareAnalysisFragment} />
+              </Grid>
+              <Grid item={true} xs={6} style={{ paddingTop: 10 }}>
+                  <StixDomainObjectOverview stixDomainObject={malwareAnalysisFragment} />
+              </Grid>
+          </Grid>
+          <Grid
+              container={true}
+              spacing={3}
+              classes={{ container: classes.gridContainer }}
+              style={{ marginTop: 25 }}
+          >
+              <Grid item={true} xs={6}>
+                  <SimpleStixObjectOrStixRelationshipStixCoreRelationships
+                      stixObjectOrStixRelationshipId={malwareAnalysisFragment.id}
+                      stixObjectOrStixRelationshipLink={`/dashboard/analysis/malware_analyses/${malwareAnalysisFragment.id}/knowledge`}
+                  />
+              </Grid>
+              <Grid item={true} xs={6}>
+                  <StixCoreObjectOrStixCoreRelationshipLastReports
+                      stixCoreObjectOrStixCoreRelationshipId={malwareAnalysisFragment.id}
+                  />
+              </Grid>
+          </Grid>
+          <Grid
+              container={true}
+              spacing={3}
+              classes={{ container: classes.gridContainer }}
+              style={{ marginTop: 25 }}
+          >
+              <Grid item={true} xs={6}>
+                  <StixCoreObjectExternalReferences stixCoreObjectId={malwareAnalysisFragment.id} />
+              </Grid>
+              <Grid item={true} xs={6}>
+                  <StixCoreObjectLatestHistory stixCoreObjectId={malwareAnalysisFragment.id} />
+              </Grid>
+          </Grid>
+          <StixCoreObjectOrStixCoreRelationshipNotes
+              stixCoreObjectOrStixCoreRelationshipId={malwareAnalysisFragment.id}
+              defaultMarking={(malwareAnalysisFragment.objectMarking?.edges ?? []).map((edge) => edge.node)}
```

Comment:
Not related to your task but could you update the component to add the following:

1. DefaultMarking becomes DefaultMarkings
2. DefaultMarkings is NOT mandatory
3. Update the toOptions method to handle the case where defaultMarkings is undefined
4. Update every component using StixCoreObjectOrStixCoreRelationshipNotes & Co, to have defaultMarkings now

Reply:
Here is the commit with the changes: https://github.com/OpenCTI-Platform/opencti/pull/3177/commits/f7b6a3025b99a155dceae5b03ffdc247485a61c5
Can you check it, please?
Thanks!